### ISSUE TITLE
String resources missing since 5.7.0 SDK update changes

### ISSUE THREAD

**Issue by jeryini** [dji-sdk/Mobile-SDK-Android-V5#328]
Hello @dji-dev @dji-lyt !

There are quite a few string resources missing for the widgets that were introduced in the 5.7.0 update. Is it possible you can include them in the 5.9 update? Also some are actually there, but the reference string does not prefix with `uxsdk_` that's why they are still marked with missing.

Also mentioned in:
* https://github.com/dji-sdk/Mobile-SDK-Android-V5/issues/260
* https://github.com/dji-sdk/Mobile-SDK-Android-V5/issues/210

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/107177'>Zendesk ticket #107177</a>:<br><br>Are you referring to the components using internal resources of the SDK? In version 5.8, some resources have been placed in the UXSDK library. In version 5.9, the remaining resources should also be placed in the UXSDK library.
<br><p><em><small>°°°</small></em></p>

**Comment by jeryini** [comment]
Do you mean, the string resources being referenced have actually moved to DJI SDK?

**Comment by dji-dev** [comment]
Agent comment from yating.liao in <a href='https://djisdksupport.zendesk.com/agent/tickets/107177'>Zendesk ticket #107177</a>:<br><br>I mean the resources you couldn't find before might have been moved inside the SDK. In version 5.8, we relocated some resources to the sample code so you can access them directly. The remaining parts will be included in the sample code for version 5.9.

What specific resources are you referring to as missing?
<br><p><em><small>°°°</small></em></p>

**Comment by AMagill** [comment]
I believe I'm seeing this same problem as well.  These are the errors I'm seeing:
```
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_gimbal_fine_tune.xml","position":{"startLine":104}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_gimbal_fine_tune.xml:105: AAPT: error: resource string/app_done (aka dji.v5.ux.test:string/app_done) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_gimbal_setting.xml","position":{"startLine":53}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_gimbal_setting.xml:54: AAPT: error: resource string/setting_menu_title_gimbal_fine_tune (aka dji.v5.ux.test:string/setting_menu_title_gimbal_fine_tune) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml","position":{"startLine":14}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml:15: AAPT: error: resource string/setting_ui_hd_sdr_custom_selected_tip (aka dji.v5.ux.test:string/setting_ui_hd_sdr_custom_selected_tip) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml","position":{"startLine":129}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml:130: AAPT: error: resource string/setting_ui_hd_sdr_mincolor_desc (aka dji.v5.ux.test:string/setting_ui_hd_sdr_mincolor_desc) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml","position":{"startLine":143}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml:144: AAPT: error: resource string/setting_ui_hd_sdr_maxcolor_desc (aka dji.v5.ux.test:string/setting_ui_hd_sdr_maxcolor_desc) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml","position":{"startLine":154}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_sdr_freq_view.xml:155: AAPT: error: resource string/setting_ui_hd_sdr_channel_select_tip (aka dji.v5.ux.test:string/setting_ui_hd_sdr_channel_select_tip) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_battery_widget.xml","position":{"startLine":33}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_battery_widget.xml:34: AAPT: error: resource string/checklist_manual_drone_battery_title (aka dji.v5.ux.test:string/checklist_manual_drone_battery_title) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml","position":{"startLine":19}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml:20: AAPT: error: resource string/setting_menu_common_aircraft_version (aka dji.v5.ux.test:string/setting_menu_common_aircraft_version) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml","position":{"startLine":25}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml:26: AAPT: error: resource string/setting_menu_common_rc_version (aka dji.v5.ux.test:string/setting_menu_common_rc_version) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml","position":{"startLine":69}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml:70: AAPT: error: resource string/setting_menu_common_flyc_serial (aka dji.v5.ux.test:string/setting_menu_common_flyc_serial) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml","position":{"startLine":76}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml:77: AAPT: error: resource string/not_a_num (aka dji.v5.ux.test:string/not_a_num) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml","position":{"startLine":76}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml:77: AAPT: error: resource string/setting_menu_common_rc_serial (aka dji.v5.ux.test:string/setting_menu_common_rc_serial) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml","position":{"startLine":83}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_about.xml:84: AAPT: error: resource string/setting_menu_common_rtk_module_serial (aka dji.v5.ux.test:string/setting_menu_common_rtk_module_serial) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_aircraft_go_home_mode_layout.xml","position":{"startLine":13}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_aircraft_go_home_mode_layout.xml:14: AAPT: error: resource string/setting_menu_flyc_rth_mode (aka dji.v5.ux.test:string/setting_menu_flyc_rth_mode) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_aircraft_go_home_mode_layout.xml","position":{"startLine":26}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_aircraft_go_home_mode_layout.xml:27: AAPT: error: resource string/setting_menu_flyc_smart_rth_set_altitude (aka dji.v5.ux.test:string/setting_menu_flyc_smart_rth_set_altitude) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_common_device_rename.xml","position":{"startLine":21}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_common_device_rename.xml:22: AAPT: error: resource string/setting_common_device_name (aka dji.v5.ux.test:string/setting_common_device_name) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_common_device_rename.xml","position":{"startLine":45}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_common_device_rename.xml:46: AAPT: error: resource string/setting_common_save (aka dji.v5.ux.test:string/setting_common_save) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_compass_status_layout.xml","position":{"startLine":40}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_compass_status_layout.xml:41: AAPT: error: resource string/setting_ui_redundancy_sensor_compass_vector (aka dji.v5.ux.test:string/setting_ui_redundancy_sensor_compass_vector) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_frequency_tab_select_layout.xml","position":{"startLine":13}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_frequency_tab_select_layout.xml:14: AAPT: error: resource string/setting_menu_title_hd_frequency (aka dji.v5.ux.test:string/setting_menu_title_hd_frequency) not found.\n    ","tool":"AAPT"}
AGPBI: {"kind":"error","text":"Android resource linking failed","sources":[{"file":"R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_omni_apas_layout.xml","position":{"startLine":16}}],"original":"ERROR: R:\\dji\\Mobile-SDK-Android-V5\\SampleCode-V5\\android-sdk-v5-uxsdk\\build\\intermediates\\packaged_res\\debug\\packageDebugResources\\layout\\uxsdk_setting_menu_omni_apas_layout.xml:17: AAPT: error: resource string/fpv_setting_safe_obstacle_avoidance_behavior_list (aka dji.v5.ux.test:string/fpv_setting_safe_obstacle_avoidance_behavior_list) not found.\n    ","tool":"AAPT"}
```

I have been able to build successfully by changing the `deps.aircraft` dependency in build.gradle (:uxsdk) from `compileOnly` to `implementation`.
